test/utils/deleteRoles.js

```diff
@@ -0,0 +1,25 @@
+const firestore = require("../../utils/firestore");
+const userCollection = firestore.collection("users");
+
+/**
+ * @return {boolean} success - are roles deleted or not
+ */
+module.exports = async (userId = null, rolesToBeDeleted = [], deleteRolesObject = false) => {
```

Comment:
Is `deleteRolesObject` an object or a boolean?

Reply:
As discussed, I have broken this util into 2 separate utils : `deleteRoles` and `deleteRolesObject`

Reply:
Nit: `shouldDeleteRoles` @samarpan1738